provider/src/main/scala/com/actian/spark_vector/provider/Main.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2016 Actian Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.actian.spark_vector.provider
+
+import java.nio.channels.ServerSocketChannel
+
+import org.apache.spark.{ Logging, SparkConf, SparkContext }
+import org.apache.spark.sql.hive.HiveContext
+
+import resource.managed
+import java.net.InetAddress
+
+object Main extends App with Logging {
+  import ProviderAuth._
+
+  private val conf = new SparkConf()
+    .setAppName("Spark-Vector external tables provider")
+    .set("spark.task.maxFailures", "1")
```

Comment:
I don't think this param will change any time soon. It is set to ensure that Spark doesn't try to retry tasks while loading into Vector (since Vector isn't able to restart tasks very easily now)